websurfx/config.lua

```diff
@@ -19,3 +19,8 @@ theme = "simple" -- the theme name which should be used for the website
 
 -- Caching
 redis_connection_url = "redis://127.0.0.1:8082" -- redis connection url address on which the client should connect on.
+
+-- Aggregator
+aggregator = {
+  random_delay = false, -- whether to add random delay before sending the request to the search engine
```

Comment:
As of now, we are only deploying the tool locally, so by default, I have set the `random_delay` parameter to false. However, once we decide to deploy it publicly, we can create separate configurations for production and local environments